source/text/implementation/vss-string_vectors.adb

```diff
@@ -32,6 +34,26 @@ package body VSS.String_Vectors is
       VSS.Implementation.String_Vectors.Reference (Self.Data);
    end Adjust;
 
+   -------------
+   -- Element --
+   -------------
+
+   function Element
+     (Self  : Virtual_String_Vector'Class;
+      Index : Positive) return VSS.Strings.Virtual_String
+   is
+      use type VSS.Implementation.String_Vectors.String_Vector_Data_Access;
+
+   begin
+      if Self.Data /= null and then Index <= Self.Data.Last then
+         return
+           VSS.Strings.Internals.To_Virtual_String (Self.Data.Data (Index));
+
+      else
+         return VSS.Strings.Empty_Virtual_String;
```

Comment:
Shall we raise Constraint_Error here?

Reply:
An idea to don't raise exceptions when it is not strongly necessary.